packages/xod-client/src/workers/run.js

```diff
@@ -0,0 +1,27 @@
+import * as R from 'ramda';
+import { noop } from 'xod-func-tools';
+import { prepareModuleOptions } from 'xod-cloud-tabtest';
+
+import WasmWorker from './wasm.worker';
```

Comment:
Why `new.module.naming`? Should it be `wasmWorker` instead? I can’t understand why does it even work? You don’t export anything `wasm.worker.js` :confused: 

What am I missing?

Reply:
New dependency `worker-loader` takes care about it

Reply:
About naming (`wasm.worker.js`, if you mean that), it points that file is not a js module like others, this is js for the web worker. And it will be loaded using `worker-loader`.

Reply:
Ok, it’s clear now. Would you pass the comment from the top of `wasm.worker.js` through Grammarly? It contains too many mistakes to communicate the idea.